src/lib.rs

```diff
@@ -32,3 +30,47 @@ pub mod ephemeral_instance_keys {
         pub static ref RANDOM_SEED: Seed = Rng::gen(&mut ThreadRng::default());
     }
 }
+
+pub mod signals {
```

Comment:
Hi @WarmBeer, why don't you add a new `signals` mod.rs file? I think `lib` mod tends to be a place to put things that you do not have a name for yet :-), but it this case you have the name.